features/bootstrap/JoindInApiContext.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+use GuzzleHttp\Client;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Webmozart\Assert\Assert;
+
+class JoindInApiContext extends BaseContext
+{
+    use FixturesTrait;
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
+    /** @var string|null */
+    private $raffleId;
```

Comment:
Not sure where this variable is used in this context? Does JoindInApiContext need to be aware of our raffler system?

Reply:
We still need the ID, it's used in the URL